mandatory/4-tax.js

```diff
@@ -13,11 +13,12 @@ function calculateSalesTax() {}
   The business has informed you that prices must have 2 decimal places
   They must also start with the currency symbol
   Write a function that adds tax to a number, and then transforms the total into the format £0.00
-
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(price) {
+  return price + (price * 0.2)
```

Comment:
This solution is incomplete. The returned value should be a string that starts with the currency symbol and the number is rounded to 2 decimal places.